packages/terra-application-layout/tests/jest/__snapshots__/DevSiteRoutingMenu.test.jsx.snap

```diff
@@ -0,0 +1,52 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Looks like this should have been removed from this PR when you made changes to migrate the DevSiteRouting Menu to the terra-dev-site repo